definitions/procedures/sync_plans.rb

```diff
@@ -0,0 +1,34 @@
+class Procedures::SyncPlans < ForemanMaintain::Procedure
+  metadata do
+    for_feature :sync_plans
+    description 'enable/disable sync plans before/after upgrade'
+  end
+
+  attr_reader :rollback
+
+  def initialize(options = {})
+    @rollback = options[:rollback]
+  end
+
+  def run
+    if rollback
+      enabled_sync_plans
+    else
+      disable_all_enabled_sync_plans
+    end
+  end
+
+  private
+
+  def disable_all_enabled_sync_plans
+    with_spinner('disabling all sync plans') do |spinner|
```

Comment:
In disable scenario, ids of enabled sync_plans are fetched through sql query which may takes sometime. That's why I added spinner there. 
In enable scenario, ids are read from file only. 
This is the only reason, I added spinner for disable but not for enable.

Please let me know if any changes required in it.